.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish pypi package
+on:
+  push:
+    tags:
+     - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+    - name: Build python package
+      run: |
+        echo ${{ github.ref_name }} > VERSION
```

Comment:
Create a temporary `VERSION` file to read from setup.py. It makes automated to specify the package version using git tag name.